cutten-web/package-lock.json

```diff
@@ -0,0 +1,11024 @@
+{
```

Comment:
delete this. not needed because we are using yarn. you will need to run `yarn install` to make sure the `yarn-lock.json` is updated instead.